polyglot/piranha/src/models/piranha_arguments.rs

```diff
@@ -141,6 +145,7 @@ impl Default for PiranhaArguments {
       number_of_ancestors_in_parent_scope: 4,
       cleanup_comments_buffer: 2,
       cleanup_comments: false,
+      dry_run: true,
```

Comment:
```suggestion
      dry_run: false,
```
Can we have default value for `dry_run` as `false`

Reply:
Like I was thinking by default the tool should do changes (in case the developer does not pass dry run flag)